students/PeterB/Session3/slicing.py

```diff
@@ -0,0 +1,36 @@
+# slicing
+# first and last
+seq = '3, 2, 1'
+def exchange_first_last(seq):
+    first = seq[-1]
+    mid = seq[1:-1]
+    last = seq[0]
+    return print([first] + mid + [last])
```

Comment:
You don't need to add the brackets, this will make it a list which you don't need. You can just have first + mid  + last